src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String CHARACTER = ",";
+    private static final String RESULT = "result";
+
+    private String readAndSortFile(String fileName) {
+        File file = new File(fileName);
+        StringBuilder builder = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                String[] splitArray = value.split(CHARACTER);
+                if (splitArray[0].equals(BUY)) {
```

Comment:
Indexes should always have constants 

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file